### PR TITLE
Add Chocolatey package.

### PR DESCRIPTION
**Pull request by juju2112**
Add files for Chocolatey package. 

Tested with:
https://github.com/chocolatey-community/chocolatey-test-environment

Once this PR is merged, I will release `1.1.0` of the sidecar, then I'll publish the Chocolatey package.

### CODE REVIEW

**Comment by juju2112** [comment]
@mpfz0r Before I release `1.0.3` of the sidecar, does the documentation need to be updated to show the minimim Graylog server compatible with Beats >7.0?

https://github.com/Graylog2/collector-sidecar/pull/391

**Comment by mpfz0r** [comment]
> @mpfz0r Before I release `1.0.3` of the Sidecar, does the documentation need to be updated to show the minimim Graylog server compatible with Beats >7.0?
> 
> #391

I guess so. They need to be running Graylog `3.2.5` or newer

**Comment by mpfz0r** [comment]
speaking of a new release: have you seen https://github.com/Graylog2/collector-sidecar/issues/401

**Comment by juju2112** [comment]
> speaking of a new release: have you seen #401

I saw it. I haven't used Go enough to understand the scope. I can go read up on it and try to get it done, though! Gives me a chance to use it.

File: dist/chocolatey/graylog-sidecar.nuspec [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+    <authors>Donald Morton</authors>
+    <projectUrl>https://github.com/Graylog2/collector-sidecar</projectUrl>
+    <iconUrl>http://cdn.rawgit.com/collector-sidecar/master/images/graylog-icon.svg</iconUrl>
+    <licenseUrl>https://github.com/Graylog2/collector-sidecar/blob/master/LICENSE</licenseUrl>
+    <requireLicenseAcceptance>true</requireLicenseAcceptance>
+    <projectSourceUrl>https://github.com/Graylog2/collector-sidecar</projectSourceUrl>
+    <docsUrl>https://docs.graylog.org/en/latest/pages/sidecar.html</docsUrl>
+    <bugTrackerUrl>https://github.com/Graylog2/collector-sidecar/issues</bugTrackerUrl>
+    <tags>graylog graylog-sidecar log</tags>
+    <summary>Manage log collectors through Graylog </summary>
```
Comment (mpfz0r) [inline dist/chocolatey/graylog-sidecar.nuspec:17]:
```suggestion
    <summary>Manage log collectors through Graylog</summary>
```

File: version.mk [hunk 2]
```diff
@@ -1,3 +1,3 @@
-COLLECTOR_VERSION = 1.1.0
+COLLECTOR_VERSION = 1.0.3
```
Comment (mpfz0r) [inline version.mk:1]: I think we should leave that at 1.1.0 in `master`
If we make a `1.1` release, we would create a `1.1` branch based on current `master` and then bump it there

**Review by mpfz0r — CHANGES_REQUESTED** [review]
A few smaller things. Rest looks good, from what I can tell.

File: version.mk (hunk 2 above)
Comment (juju2112) [reply version.mk:1]: Yeah, I was confused on that. I've got it now!

**Pull request merged**